packages/appstore/src/constants/wallets-tour-steps-config.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Step, Locale, Styles } from 'react-joyride';
+import { Text, SpanButton, Icon } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import 'Components/toggle-account-type/toggle-account-type.scss';
+
+export const getWalletStepConfig = (): Step[] => [
+    {
+        title: (
+            <React.Fragment>
+                <Text as='p' weight='bold' color='brand-red-coral'>
+                    {localize('Wallets')}
+                </Text>
+                <div className='toggle-account-type__divider' />
+            </React.Fragment>
+        ),
+        content: (
+            <Text as='p' size='xs'>
+                {localize('This is your Wallet. You can see your total balance for this Wallet here.')}
+            </Text>
+        ),
+        target: '.wallet-header',
+        disableBeacon: true,
+        disableOverlayClose: true,
+        placement: 'bottom-end',
+        hideBackButton: true,
+        spotlightPadding: 0,
+        styles: {
+            spotlight: {
+                width: '115.2rem',
+                height: '8rem',
+                borderRadius: '1.6rem 1.6rem 0rem 0rem',
+            },
+        },
+    },
+    {
+        title: (
+            <React.Fragment>
+                <Text as='p' weight='bold' color='brand-red-coral'>
+                    {localize('Expand Wallets')}
+                </Text>
+                <div className='toggle-account-type__divider' />
+            </React.Fragment>
+        ),
+        content: (
+            <Text as='p' size='xs'>
+                <Localize i18n_default_text='Click here to expand or collapse your Wallet and view the trading accounts linked to this Wallet.' />
+            </Text>
+        ),
+
```

Comment:
@aizad-deriv you can add a comment here to clear the selection will apply to the first element which has this class for the further usege